contrib/builder/deb/generate.sh

```diff
@@ -55,6 +55,7 @@ for version in "${versions[@]}"; do
 		libapparmor-dev # for "sys/apparmor.h"
 		libdevmapper-dev # for "libdevmapper.h"
 		libsqlite3-dev # for "sqlite3.h"
+		libsystemd-journal-dev # for "sd-journal.h" and libraries
```

Comment:
In that case, maybe we'd be better off with a `case` statement to add this one specifically?  ie, something like:
```bash
case "$suite" in
    precise) ;;
    stretch|sid) packages+=( libsystemd-dev ) ;;
    *) packages+=( libsystemd-journal-dev ) ;;
esac
```